src/Sulu/Bundle/MediaBundle/Tests/Features/collection_management.feature

```diff
@@ -42,6 +45,22 @@ Feature: Collection management
         And I am on the media collection edit page
         Then I expect to see "4" ".item" elements
 
+    Scenario: Move collection
+        Given the media collection "Dornbirn" exists
+        Given the media collection "Foobar" exists
```

Comment:
You should never repeat "Given". Should read

````
Given the media collection ".." exists
And the media collection ".." exists
When I click the toolbar item "..."
And I expect an overlay to appear
...
Then I expect to see "1" ".data-navigation-item" elements
````

i.e. `Given` should start a section which defines the context. `When` should be the section about the actions you perform. `Then` should be what you expect to happen.

Basically the scenario should make sense when you read it. We need to improve this generally.